File: AppBanHangReactNative/src/screens/about/AboutUsScreen.js
import React from 'react';
import { View, Text, StyleSheet, ScrollView } from 'react-native';
import { useTranslation } from 'react-i18next';
import { useTheme } from '../../context/ThemeContext';

const AboutUsScreen = () => {
  const { t } = useTranslation();
  const { theme } = useTheme();
  
  return (
    <ScrollView style={[styles.container, { backgroundColor: theme.backgroundColor }]}>
      <View style={styles.section}>
        <Text style={[styles.title, { color: theme.textColor }]}>{t('aboutUs.title')}</Text>
        <Text style={[styles.description, { color: theme.textColor }]}>{t('aboutUs.description')}</Text>
      </View>

      <View style={styles.section}>
        <Text style={[styles.sectionText, { color: theme.textColor }]}>{t('aboutUs.content.mission')}</Text>
        <Text style={[styles.sectionText, { color: theme.textColor }]}>{t('aboutUs.content.vision')}</Text>
        <Text style={[styles.sectionText, { color: theme.textColor }]}>{t('aboutUs.content.story')}</Text>
      </View>

      <View style={styles.section}>
        <Text style={[styles.featureTitle, { color: theme.textColor }]}>Features:</Text>
        <Text style={[styles.feature, { color: theme.textColor }]}>• {t('aboutUs.features.variety')}</Text>
        <Text style={[styles.feature, { color: theme.textColor }]}>• {t('aboutUs.features.service')}</Text>
        <Text style={[styles.feature, { color: theme.textColor }]}>• {t('aboutUs.features.delivery')}</Text>
        <Text style={[styles.feature, { color: theme.textColor }]}>• {t('aboutUs.features.payment')}</Text>
      </View>
    </ScrollView>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    padding: 20,
  },
  section: {
    marginBottom: 20,
  },
  title: {
    fontSize: 24,
    fontWeight: 'bold',
    marginBottom: 10,
  },
  description: {
    fontSize: 16,
    marginBottom: 20,
  },
  sectionText: {
    fontSize: 16,
    marginBottom: 10,
  },
  featureTitle: {
    fontSize: 18,
    fontWeight: 'bold',
    marginBottom: 10,
  },
  feature: {
    fontSize: 16,
    marginBottom: 5,
    marginLeft: 10,
  },
});

export default AboutUsScreen;

File: app_banhang2/lib/screens/about_us_screen.dart
import 'package:flutter/material.dart';
import 'package:easy_localization/easy_localization.dart';

class AboutUsScreen extends StatelessWidget {
  const AboutUsScreen({super.key});

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('aboutUs.title'.tr()),
      ),
      body: SingleChildScrollView(
        padding: const EdgeInsets.all(20),
        child: Column(
          crossAxisAlignment: CrossAxisAlignment.start,
          children: [
            Text(
              'aboutUs.description'.tr(),
              style: const TextStyle(fontSize: 16),
            ),
            const SizedBox(height: 20),
            
            // Mission, Vision, Story section
            _buildText(context, 'aboutUs.content.mission'.tr()),
            _buildText(context, 'aboutUs.content.vision'.tr()),
            _buildText(context, 'aboutUs.content.story'.tr()),
            const SizedBox(height: 20),
            
            // Features section
            Text(
              'Features:',
              style: const TextStyle(
                fontSize: 18,
                fontWeight: FontWeight.bold,
              ),
            ),
            const SizedBox(height: 10),
            
            // Feature items
            _buildFeatureItem('aboutUs.features.variety'.tr()),
            _buildFeatureItem('aboutUs.features.service'.tr()),
            _buildFeatureItem('aboutUs.features.delivery'.tr()),
            _buildFeatureItem('aboutUs.features.payment'.tr()),
          ],
        ),
      ),
    );
  }

  Widget _buildText(BuildContext context, String text) {
    return Padding(
      padding: const EdgeInsets.only(bottom: 10),
      child: Text(
        text,
        style: const TextStyle(fontSize: 16),
      ),
    );
  }

  Widget _buildFeatureItem(String text) {
    return Padding(
      padding: const EdgeInsets.only(left: 10, bottom: 5),
      child: Row(
        crossAxisAlignment: CrossAxisAlignment.start,
        children: [
          const Text('• ', style: TextStyle(fontSize: 16)),
          Expanded(
            child: Text(text, style: const TextStyle(fontSize: 16)),
          ),
        ],
      ),
    );
  }
}